.rubocop.yml

```diff
@@ -0,0 +1,508 @@
+require: rubocop-rspec
```

Comment:
Yeah, that will be an interesting discussion. In the end I don't really care about the list, but am more interested in getting style discussions out of the way. Machines are way better at enforcing consistency....